DEVELOPERS.md

```diff
@@ -536,31 +539,31 @@ job-server. Refer to the documentation of
 
 This interacts with the [JobRequestAPIList](jobserver/api/jobs.py#L239) (GET /job-requests/) view for reading JobRequests, and the [JobAPIUpdate](jobserver/api/jobs.py#L54) (POST /jobs/) view for updating the Jobs table. (Current as of 2024-09.)
 
-### [Airlock](https://github.com/opensafely-core/airlock)
+### [airlock](https://github.com/opensafely-core/airlock)
 
-[Airlock](https://github.com/opensafely-core/airlock) is the container that
-[OutputCheckers](jobserver/authorization/roles.py#63) interact with to check
+[airlock](https://github.com/opensafely-core/airlock) is the container that
+[OutputCheckers](jobserver/authorization/roles.py#L63) interact with to check
```

Comment:
- "fixing" some links to specific lines (which, in the next commit, I realise are a bad idea altogether)